debugger-launchjson.md

```diff
@@ -0,0 +1,110 @@
+# Configurating launch.json for C# debugging
+The launch.json file is used to configure the debugger in Visual Studio Code.
+
+Visual Studio Code generates a launch.json with almost all of the required information. 
+If your workspace has only one launchable project, the C# extension will offer to automatically generate this file. 
+If you missed this prompt, you can force the generation by executing the command `.NET: Generate Assets for Build and Debug` from the VS Code command pallet.
+The generated file contains two sections. One that configures debugging for launch and a second that configures debugging for attach.
+
+If you have more than one launchable project, then you will need to modify your launch.json file by hand. 
+Visual Studio Code will still generate a basic template, but you will need to fill in the 'program' field to point at the executable dll that you would like to debug.
+
+
+# Configurating VS Code's debugging behavior
+
+## PreLaunchTask
+The `preLaunchTask` field runs the associated taskName in tasks.json before debugging your program. You can get the default build prelaunch task by executing the command `Tasks: Configure Tasks Runner` from the VS Code command pallet.
+
+This will create a task that runs `dotnet build`. You can read more about tasks at [https://code.visualstudio.com/docs/editor/tasks](https://code.visualstudio.com/docs/editor/tasks).
+
+## Program
+The program field is set to the path to the application dll or .NET Core host executable to launch. 
+
+Example: "${workspaceRoot}/bin/Debug/\<target-framework\>/\<project-name.dll\>" where:
+
+    <target-framework>: (example: 'netstandard1.5') This is the framework that the app is being built for. It is set in the project.json file.
```

Comment:
>     [](start = 0, length = 4)

I think you don't want these indents as that puts this in a code block, and I don't think you want that.